build.gradle.kts

```diff
@@ -18,29 +18,14 @@ plugins{
     id("io.github.gradle-nexus.publish-plugin") version "2.0.0"
 }
 
-def swoVersion = property('swo.agent.version')
+val swoAgentVersion = "2.10.0" // when updating also update the one in :dependencyManagement
```

Comment:
Hmm kind of a shame that there's this possible gotcha now. I guess it's somehow not feasible to get this value from dependencyManagement?

Reply:
probably should just leave them in the `gradle.properties` file 🤔